app/src/main/java/project/ProjectWindow.java

```diff
@@ -131,21 +144,30 @@ private JPanel createFiltersPanel() {
         gbc.weighty = 1.0; // This spacer takes up extra vertical space
         filtersPanel.add(Box.createVerticalGlue(), gbc);
 
-        // Row 5: Export Button at the bottom
+        // Row 5: Export Button (CSV) at the bottom
         gbc.gridy = 5;
         gbc.weighty = 0; // Reset vertical weight for the button
         gbc.insets = new Insets(10, 10, 20, 10); // Extra space below the export button
         gbc.anchor = GridBagConstraints.SOUTH; // Anchor at the bottom
-        BetterButton exportButton = new BetterButton("Export");
+        BetterButton exportButton = new BetterButton("Export as CSV");
         filtersPanel.add(exportButton, gbc);
-        exportButton.addActionListener(e -> exportData());
+        exportButton.addActionListener(e -> exportData("exported_data.csv"));
+
+        // Row 6: Export Button (JSON) at the bottom
+        gbc.gridy = 6;
+        gbc.weighty = 0; // Reset vertical weight for the button
+        gbc.insets = new Insets(10, 10, 20, 10); // Extra space below the export button
+        gbc.anchor = GridBagConstraints.SOUTH; // Anchor at the bottom
+        BetterButton exportButton2 = new BetterButton("Export as JSON");
+        filtersPanel.add(exportButton2, gbc);
+        exportButton2.addActionListener(e -> exportData("exported_data.json"));
```

Comment:
actual logic for saving to json is needed. json is data object file, we can't just use the same write out and only change file extensions that's not how it works. Regardless, I did have a json writer from a previous project that I will be including so no worries here.